apps/native-component-list/src/screens/UI/DateTimePickerScreen.tsx

```diff
@@ -0,0 +1,76 @@
+import { DateTimePicker, DatePickerProps } from '@expo/ui/components/DatePicker';
+import { Picker } from '@expo/ui/components/Picker';
+import * as React from 'react';
+import { Platform, ScrollView, Text, View } from 'react-native';
+
+import { Page, Section } from '../../components/Page';
+
+export default function DatePickerScreen() {
+  const [selectedDate, setSelectedDate] = React.useState(new Date());
+
+  const displayOptions =
+    Platform.OS === 'ios' ? ['compact', 'graphical', 'wheel'] : ['picker', 'input'];
+  const [selectedIndex, setSelectedIndex] = React.useState(0);
+
+  const typeOptions = ['date', 'hourAndMinute', 'dateAndTime'];
+  const [typeIndex, setTypeIndex] = React.useState(0);
+
+  function getPickerType() {
+    const str = displayOptions[selectedIndex];
+    return `${str.charAt(0).toUpperCase()}${str.slice(1)} picker`;
+  }
+
+  return (
+    <ScrollView>
+      <Page>
+        <Section title="Selected Date">
+          <Text>{selectedDate.toDateString()}</Text>
+        </Section>
+        <Section title="Selected Time">
+          <Text>{selectedDate.toTimeString()}</Text>
+        </Section>
+        <Section title={getPickerType()}>
+          <View style={{ gap: 20 }}>
+            <DateTimePicker
+              title=""
```

Comment:
Do we have to pass title, even if it's empty?

Reply:
Can we move it to the native code? It looks weird from the user perspective

Reply:
It's already there, we default to an empty string. The TS type is optional. I was just using it in the example screen for testing, I'll remove it 👍